website/versioned_docs/version-0.5/navigation.md

```diff
@@ -12,22 +12,86 @@ If you'd like to achieve a native look and feel on both Android and iOS, or you'
 
 ## React Navigation
 
-The community solution to navigation is a standalone library that allows developers to set up the screens of an app with a few lines of code.
+The community solution to navigation is a standalone library that allows developers to set up the screens of an app with a few lines of code. Assuming you are on the latest react-native version.
 
 The first step is to install in your project:
 
 ```
-npm install --save react-navigation
+yarn add react-navigation
+# or with npm
+# npm install react-navigation
 ```
 
-The second step is to install react-native-gesture-handler
+The second step is to install additional dependencies such as `react-native-gesture-handler`, `react-native-reanimated`, and `react-native-screens`:
 
 ```
 yarn add react-native-gesture-handler
 # or with npm
 # npm install --save react-native-gesture-handler
 ```
 
+```
+yarn add react-native-reanimated
+# or with npm
+# npm install --save react-native-reanimated
+```
+
+```
+yarn add react-native-screens
```

Comment:
can you join then into one command like: 
`npm install react-native-reanimated react-native-gesture-handler react-native-screens react-native-safe-area-context @react-native-community/masked-view`.

Also, we in react-navigation are now not supportive of using `yarn`, so If you want, feel free to mention only `npm` (nit).